README.md

```diff
@@ -11,7 +11,13 @@ GitHub Action for running [`cargo-deny`](https://github.com/EmbarkStudios/cargo-
 Create a `deny.toml` file in the root of the repo to use as rules for the action ([example](https://github.com/EmbarkStudios/cargo-deny/blob/master/deny.toml)).
 See [`cargo-deny`](https://github.com/EmbarkStudios/cargo-deny) for instructions and details of the format and capabilities.
 
-This action will run `cargo-deny check` and report failure if any banned crates or disallowed open souce licenses are found used in the crate or its dependencies.
+This action will run `cargo-deny check` and report failure if any banned crates or disallowed open source licenses are found used in the crate or its dependencies.
+
+The action has three optional inputs
+
+* `log-level`: The log level to use for `cargo-deny`, default is `warn`
+* `command`: The command to use for `cargo-deny`, default is `check`
+* `arguments`: The argument to pass to `cargo-deny`, default is `--all-features`
```

Comment:
```suggestion
* `arguments`: The argument to pass to `cargo-deny`, default is `--all-features`. See [Common Options](https://embarkstudios.github.io/cargo-deny/cli/common.html) for a list of the available options.
```

Reply:
Good point. I'm a newbie to cargo-deny also, so that would have helped me as well.